packages/components/src/Link/Link.stories.tsx

```diff
@@ -39,7 +38,7 @@ const defaultArgs = {
   children: "Link",
   variant: "link" as const,
   color: "brand" as const,
-  href: "",
+  href: "www.example.com",
```

Comment:
Randomly gave the link examples an href. Looked wrong having an empty string 🧐

Reply:
I like to use an empty string because you can click the component to see how it looks when you interact with it without being sent to another page.

Reply:
Makes sense

Reply:
Changing it back. I did notice that the browser will still redirect to another page. With an empty href (or #) I still get redirect to the current story's documentation page... which is weird.

Not sure if there's some browser thing I don't understand, or if Storybook is now intercepting link clicks or something (maybe that's likely).

Reply:
Oh yeah, it does. Maybe I should put `event.preventDefault()` on them..